src/garage/np/algos/meta_rl_algorithm.py

```diff
@@ -0,0 +1,35 @@
+"""Interface of Meta-RL ALgorithms."""
+import abc
+
+from garage.np.algos import RLAlgorithm
+
+
+class MetaRLAlgorithm(RLAlgorithm, abc.ABC):
+    """Base class for Meta-RL Algorithms."""
+
+    @abc.abstractmethod
+    def get_exploration_policy(self):
+        """Return a policy used before adaptation to a specific task.
+
+        Each time it is retrieved, this policy should only be evaluated in one
+        task.
+
+        Returns:
+            garage.Policy: The "un-adapted" policy.
+
+        """
+
+    @abc.abstractmethod
+    def adapt_policy(self, exploration_policy, trajectories):
+        """Produce a policy adapted for a task.
+
+        Args:
+            exploration_policy (garage.Policy): The policy used to explore the
```

Comment:
the exploration policy and pre-adaptation policy don't need to be the same. 

what shall it not be used for?
what are the chances people actually follow this rule?
are you saying this method will mutate the policy?
in the case of a naive meta-RL based continual learner (e.g. RL2), does this even apply?

Reply:
The goal here is to express that ownership of the policy returned by `get_exploration_policy()` is returned to the meta algorithm. This makes it as simple as possible for the implementer of the meta algorithm.

Reply:
okay i didn't actually get that from your docstring. this seems subtle enough that maybe it needs to be handeld by a note and maybe a short example or two.